galsim/phase_screens.py

```diff
@@ -1122,3 +1122,113 @@ def _wavefront_gradient(self, u, v, t, theta):
 class _DummyScreen(OpticalScreen):
     def _wavefront(self, *args):
         raise RuntimeError("Shouldn't reach this")
+
+
+class TableScreen(object):
```

Comment:
Aside from the .T issue, the only other concern I have with this is the class name.  "Table Screen" doesn't seem particularly self-documenting as to what the class is about.   The fact that it takes a LookupTable2D argument doesn't seem like the right core feature to put into the name.  That's really just an implementation detail about how to define the 2d function.
Other possibilities:
* UserScreen
* CustomScreen
* TwoDScreen
* ...

Reply:
Good point.  Switched to `UserScreen`.